app/src/androidTest/java/net/skyscanner/backpack/compose/BpkSnapshotTest.kt

```diff
@@ -71,7 +71,7 @@ open class BpkSnapshotTest(private val tags: List<Any> = emptyList()) : Screensh
             providers = providers,
             comparison = { name ->
                 assertion()
-
+                waitForIdle()
```

Comment:
how often did you run this to make sure it's not flaky? this was already here for other tests as well to wait for any kinds of animations/settling etc